botCommands/__snapshots__/points.test.js.snap

```diff
@@ -55,36 +55,36 @@ exports[`/leaderboard callback returns correct output 8`] = `
+exports[`<@!1233456679> ++ callback returns correct output for a user mentioning themselves 1`] = `"http://media0.giphy.com/media/RddAJiGxTPQFa/200.gif"`;
 
-exports[`@user ++ callback returns correct output for a user mentioning themselves 2`] = `"You can't do that!"`;
+exports[`<@!1233456679> ++ callback returns correct output for a user mentioning themselves 2`] = `"You can't do that!"`;
 
-exports[`@user ++ callback returns correct output for more than five mentioned users 1`] = `"you can only do 5 at a time..... "`;
+exports[`<@!1233456679> ++ callback returns correct output for more than five mentioned users 1`] = `"you can only do 5 at a time..... "`;
 
-exports[`@user ++ callback returns correct output for more than five mentioned users 2`] = `"Sweet! <@2> now has 11 points"`;
+exports[`<@!1233456679> ++ callback returns correct output for more than five mentioned users 2`] = `"Sweet! <@2> now has 11 points"`;
 
-exports[`@user ++ callback returns correct output for more than five mentioned users 3`] = `"Nice! <@2> now has 4 points"`;
+exports[`<@!1233456679> ++ callback returns correct output for more than five mentioned users 3`] = `"Nice! <@2> now has 4 points"`;
 
-exports[`@user ++ callback returns correct output for more than five mentioned users 4`] = `"Nice! <@2> now has 2 points"`;
+exports[`<@!1233456679> ++ callback returns correct output for more than five mentioned users 4`] = `"Nice! <@2> now has 2 points"`;
 
-exports[`@user ++ callback returns correct output for more than five mentioned users 5`] = `"Nice! <@2> now has 1 point"`;
+exports[`<@!1233456679> ++ callback returns correct output for more than five mentioned users 5`] = `"Nice! <@2> now has 1 point"`;
 
-exports[`@user ++ callback returns correct output for more than five mentioned users 6`] = `"Sweet! <@2> now has 22 points"`;
+exports[`<@!1233456679> ++ callback returns correct output for more than five mentioned users 6`] = `"Sweet! <@2> now has 22 points"`;
 
-exports[`@user ++ callback returns correct output for up to five mentioned users 1`] = `"Woot! <@2> now has 34 points"`;
+exports[`<@!1233456679> ++ callback returns correct output for up to five mentioned users 1`] = `"Woot! <@2> now has 34 points"`;
 
-exports[`@user ++ callback returns correct output for up to five mentioned users 2`] = `"Sweet! <@2> now has 22 points"`;
+exports[`<@!1233456679> ++ callback returns correct output for up to five mentioned users 2`] = `"Sweet! <@2> now has 22 points"`;
 
-exports[`@user ++ callback returns correct output for up to five mentioned users 3`] = `"Nice! <@2> now has 3 points"`;
+exports[`<@!1233456679> ++ callback returns correct output for up to five mentioned users 3`] = `"Nice! <@2> now has 3 points"`;
 
-exports[`@user ++ callback returns correct output for up to five mentioned users 4`] = `"Nice! <@2> now has 1 point"`;
+exports[`<@!1233456679> ++ callback returns correct output for up to five mentioned users 4`] = `"Nice! <@2> now has 1 point"`;
```

Comment:
Thanks for catching this and ensuring our snapshot tests were correct.

Reply:
It's small, but I want to change "<@!1233456679>" to something a little more uniform, like <@!123456789>, just so it's a little more predictable. My silly string was just an example. Go ahead and change it then update the snapshot tests one more time, then I will merge.

Reply:
Ok, no worries. Will do it later today :)

Reply:
Changed it :)

Reply:
Were all of the snapshots updated? In the preview it looks like some still use the old string.

Reply:
Oh whoopsie! Fixing it now

Reply:
Fixed it! Please take a look :)